src/test/modelmanager_test.cpp

```diff
@@ -25,6 +25,7 @@
 #include "../model.hpp"
 #include "../modelmanager.hpp"
 #include "../prediction_service_utils.hpp"
+#include "absl/synchronization/notification.h"
```

Comment:
Since you are using absl, please check build without MediaPipe (isnt absl dependency of MP? might be missing)